api/krusty/localizer/runner_test.go

```diff
@@ -141,9 +141,68 @@ const (
   }
 }
 `
+
+	simpleURL = "https://github.com/kubernetes-sigs/kustomize//api/krusty/testdata/localize/simple"
+
+	simpleKustomization = `apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namePrefix: localize-
+resources:
+- deployment.yaml
+- service.yaml
+`
+
+	simpleDeployment = `apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: test-deployment-simple
+  labels:
+    app: deployment-simple
+spec:
+  selector:
+    matchLabels:
+      app: simple
+  template:
+    metadata:
+      labels:
+        app: simple
+    spec:
+      containers:
+      - name: nginx
+        image: nginx:1.16
+        ports:
+        - containerPort: 8080
+`
+	simpleService = `apiVersion: v1
+kind: Service
+metadata:
+  name: test-service-simple
+spec:
+  selector:
+    app: deployment-simple
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 8080 
+`
+
+	remoteHPA = `apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: hpa-deployment
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: localize-test-deployment-simple
+  minReplicas: 1
+  maxReplicas: 10`
+
+	urlQuery = "?submodules=0&ref=kustomize/v4.5.7&timeout=300"
 )
 
-func prepareFs(t *testing.T, files map[string]string) (memoryFs filesys.FileSystem, actualFs filesys.FileSystem, testDir filesys.ConfirmedDir) {
+func prepareFs(t *testing.T, dirs []string, files map[string]string) (
```

Comment:
Since we have a lot of helper functions now, can we move these helpers to their own file `api/krusty/localizer/testutils.go`? I think it will help keep this directory organized.

Reply:
I moved all the helper functions to the top of the file for now. I'm all for improving organization. The problem with moving the helper functions to `testutils.go` is this file can't exist in package `localizer_test` because the file name doesn't end in `_test.go`. We don't want the test helper functions to be publicly exposed in the `localizer` package either. 

If moving the helper functions to the top of the file is not clean enough, I came up with a few other options:
1. Create a `loctest` package under `api/testutils` and move the helper functions to a file there called `helper.go`.
2. Create a `testutils` package under `api/krusty/localizer/internal` and move the helper functions to a file there called `helper.go`.
3. Move the helper functions to a new `utils_test.go` file under `api/krusty/localizer`.